modules/nf-core/skani/dist/main.nf

```diff
@@ -0,0 +1,50 @@
+process SKANI_DIST {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/skani:0.2.1--h4ac6f70_0':
+        'biocontainers/skani:0.2.1--h4ac6f70_0' }"
+
+    input:
+    tuple val(meta) , path(query)
+    tuple val(meta2), path(reference)
+
+    output:
+    tuple val(meta), path("${prefix}.tsv")  , emit: dist
+    path "versions.yml"                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "${meta.id}_${meta2.id}"
+    """
+    skani \\
+        dist \\
+            -q ${query} \\
+            -r ${reference} \\
```

Comment:
Should the `-rl`/`-ql` options be supported? |

```
        --ql <query list file>        File with each line containing one fasta/sketch file.
        --rl <reference list file>    File with each line containing one fasta/sketch file.

```

Might help reduce the number of parallel very small jobs if you were to do it one by one